### PR TITLE
{CI} Test auto rebase to enhance test speed

### PR DESCRIPTION
**Pull request by wangzelin007**
…ying --blob-url with --auth-mode login

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
CI

File: .azure-pipelines/templates/automation_test.yml [hunk 1]
```diff
@@ -37,16 +37,16 @@ steps:
       module="${{ parameters.module }}"
       if [[ -n $module ]]; then
         echo "Running test for module '$module'"
-        azdev test --no-exitfirst --verbose --series $module
+        azdev test --no-exitfirst --verbose --series $module -a "-n 4 --durations=10"
```
Comment (jiasli) [inline .azure-pipelines/templates/automation_test.yml:46]: `-n 4` means 4 tests can run concurrently, but many Azure CLI tests cannot be run concurrently. That's why we have that `--series` flag.

**Pull request closed without merging**